pkg/e2e/spinup_test.go

```diff
@@ -128,9 +141,9 @@ func spinup(t testing.TB, ctx context.Context, cfg config) (chan error, error) {
 				t.Logf("%s STDOUT\n %s", cmd.Path, stdout.String())
 			}
 
-			return err
+			return doneErr
 		}, func(error) {
-			cmd.Process.Signal(syscall.SIGTERM)
+			cmd.Process.Kill()
```

Comment:
For some reason with SIGTERM on Windows Kedge never got interrupted in the e2e tests.

Reply:
sigterm and sigint aren't _really_ supported by windows; this looks sensible.